apps/package.json

```diff
@@ -15,6 +15,7 @@
     "test:integration": "grunt integrationTest",
     "test:entry": "grunt karma:entry",
     "test:code-studio": "grunt codeStudioTest",
+    "flow": "flow src",
```

Comment:
we could do that. I think the bulk of the value flow provides would be in the production code, so I didn't bother enabling it for tests.